dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -594,6 +600,87 @@ private void checkToolTokens(User authUser, Long userId, String organization) {
         EntryVersionHelper.stripContent(tools, this.userDAO);
         return tools;
     }
+    @GET
+    @Path("/{userId}/organizations")
```

Comment:
This userId path param lets an admin fetch organizations for another user.

There was another PR where I followed this pattern (I forget which one), and the feedback was to remove it, to not allow admins to fetch other users' entities.

If we're continuing to follow that pattern, then we should remove the `userid`, and just have `/users/organizations`.

Reply:
> There was another PR where I followed this pattern (I forget which one), and the feedback was to remove it, to not allow admins to fetch other users' entities.

[Here](https://github.com/dockstore/dockstore/pull/2672/files#r308764386) is the discussion from that other PR, which I realize now may not be exactly the same use case.

Reply:
Unfortunately I can't seem to link to a resolved conversation, even though it let me get a link... See the resolved conversation on line 697 of UserResource.java